pkgs/by-name/ca/cargo-v5/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  dbus,
+  udev,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "cargo-v5";
+  version = "0.8.2";
+
+  src = fetchFromGitHub {
+    owner = "vexide";
+    repo = "cargo-v5";
+    rev = "9d5f6e014c80838ff2b0cca401bcbd518c1e9274";
+    hash = "sha256-bXzJvlhG/IJOu+D1iluJD3wDoiJXWzXYJ+ZUG6xMCeA=";
+  };
+
+  cargoHash = "sha256-OQ3XY8RzmjGZWFyTY3HY9AYb8qDHHeEAuWa+3WNOtRM=";
+
+  buildFeatures = [ "full" ];
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [
+    dbus
+    udev
+    openssl
+  ];
+
```

Comment:
Unfortunately this doesn't seem to be possible because you need to pass two arguments (first `v5`, then `--version`) to get the binary to print its version, and `versionCheckHook` doesn't support this.

Reply:
What's the output of `cargo-v5 --version`?

Reply:
```
error: unexpected argument '--version' found

Usage: cargo <COMMAND>

For more information, try '--help'.
```